### PR TITLE
Vimbranch

### PR DESCRIPTION
**Pull request by rishi71095**

### CODE REVIEW

File: MDSTimeManager/system/TimeManager.py [hunk 1]
```diff
@@ -56,9 +57,10 @@
 
 Courses = [Data533, Data543, Data571, Data581]
```
Comment (a-foote) [inline MDSTimeManager/system/TimeManager.py]: can we get rid of this line?

File: MDSTimeManager/system/TimeManager.py [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 import system.timemanager as tm 
 import system.deliverableviewer as dv
 import datetime 
+                  
+#######################################################################################################################################
+
+# creating Block 3 objects
 
 # Course 1
 Data533 = c.Course("Object Oriented Programming","Khalad Hasan","3",1)
```
Comment (a-foote) [inline MDSTimeManager/system/TimeManager.py]: we shouldn't need these lines (12-15), right?

File: MDSTimeManager/system/TimeManager.py [hunk 3]
```diff
@@ -56,9 +57,10 @@
 
 Courses = [Data533, Data543, Data571, Data581]  
 
+#######################################################################################################################################
+
 def userinput():
```
Comment (a-foote) [inline MDSTimeManager/system/TimeManager.py]: can we have it so the courselist is an input to this function so it's something that will get created via the main file I'm going to create

**Review by a-foote — CHANGES_REQUESTED** [review]
Please see my comments in the timemanager.py section.

File: MDSTimeManager/system/TimeManager.py (hunk 3 above)
Comment (rishi71095) [reply MDSTimeManager/system/TimeManager.py]: Courses = [Data533, Data543, Data571, Data581] is this the courselist you are talking about?

File: MDSTimeManager/system/TimeManager.py (hunk 1 above)
Comment (rishi71095) [reply MDSTimeManager/system/TimeManager.py]: I will have to look at this tomorrow. I know I have definitely used this list in more than one place, so I will have to check them all.

Comment (a-foote) [reply MDSTimeManager/system/TimeManager.py]: I am working on creating the main module that will have the object setup and this courselist line in it if that helps.

File: MDSTimeManager/system/TimeManager.py (hunk 3 above)
Comment (a-foote) [reply MDSTimeManager/system/TimeManager.py]: Yeah, I think it should be removed from the timemanager.py and can just be passed into the input() function from the main execution function I'm doing.

File: MDSTimeManager/system/TimeManager.py (hunk 2 above)
Comment (rishi71095) [reply MDSTimeManager/system/TimeManager.py:12]: This was demarcation for when the object initialization ends and  where methods start, but if we we wont have objects in tm file, then this all will be removed.

**Review by a-foote — APPROVED** [review]

**Pull request merged**